### DISCUSSION TITLE
Authentication requests not found

### DISCUSSION THREAD

**Discussion by MorayM** [community/community#109447] [Mobile]
### Select Topic Area

Bug

### Body

I have the GitHub mobile app installed on my phone (Fairphone FP5). Whenever I get prompted to authenticate something on GitHub, e.g. logging in on a new device or entering "sudo mode", the default option is to authenticate through the mobile app. I get the notification on my phone, but when I tap on it to open, I get a modal with the message "No authentication requests found" with just a close button. I've tried it on a couple of phones, and each device has the same result, every time. Using a 2FA code generator works fine.

**Comment by iskae** [comment]
Hi @MorayM 👋 

Thank you for reaching out ❤️ 

We would love to investigate this further 🙇 Can you maybe elaborate a bit more on this?

- Which version of GitHub Mobile app are you using?
- Do you have any other devices that you have GitHub Mobile installed? Do you receive notifications on your other devices?

**Reply by MorayM** [reply]
Hi, thanks for getting back to me. I'm using v1.149.1 of the Android app. I do get notifications on other devices (if they're switched on) with the same result.

Weirdly, after a couple of weeks of not working, I had a notification today which worked - first time ever!

**Reply by iskae** [reply]
Oh! Glad that it worked 😊 In this case I would assume there was a hiccup with our 2FA requests and I will investigate this further on our end 🙇 Thanks for the addition info ❤️

**Comment by maximuskowalski** [comment]
Have found that if I expand the notification in anyway to read it before clicking this happens. If I just tap the notification without unfolding or expanding in any way to read it will behave as expected.

**Reply by ohue-ruth** [reply]
I have the same issue, even without unfolding the notification. To bypass the problem I go directly to my github mobile app (without clicking on the notification) to have the code window displayed.
